### DISCUSSION TITLE
YI Home Camera 3

### DISCUSSION THREAD

**Discussion by paicl01** [roleoroleo/yi-hack-Allwinner-v2#296] [Q&A]
My model has the label "Home Camera3".
The model type is YYS.2518,
DID: IFUSY32OTB8E8210310
FW Version: 8.3.0
The app does not allow to update to a higher  FW version,, e.g. 9.x
In the compatibility list models with IFUS are listed, but only  with 9.x FW.

I tried all different hack versions on a 16 GB SD card, but never happened anything in the camera setup.
The SD card was empty in each case, after I took it out of the camera (which means that the camera does read the card).
Camera works normally with app control after all attempts. 

Is there a chance to include this model in future updates of your hack? 
Would be great!
Thank you

**Comment by roleoroleo** [comment]
https://github.com/roleoroleo/yi-hack-Allwinner

**Comment by paicl01** [comment]
after your hint I  tried again with y25ga from Allwinner project  using  a different SD card, and suddenly it works
Great, thank you very much.
